frontend/package.json

```
@@ -33,20 +34,19 @@
     "react-i18next": "^13.5.0",
     "react-router-dom": "^7.18.2",
```

Biome (2.5.5)

[error] The key react-router-dom was already declared.

(lint/suspicious/noDuplicateObjectKeys)